ranker/rankDocuments.py

```diff
@@ -50,6 +52,8 @@ def rank(queryTerms, termReverseMap, invertedIndex):
     docWeights = np.zeros(len(termReverseMap))
     for rawTerm in document['body'].lower().split():
       term = porterStemmer.stem(rawTerm)
+      if term in excludedTerms: 
+         flag = True
```

Comment:
Add a break below this flag=True b/c we don't need to process the rest of the document